packages/react/src/component-tests/IcSelect/IcSelectSearchable.cy.tsx

```diff
@@ -0,0 +1,938 @@
+      ARIA_SELECTED
+    );
+  });
+
+  // can't mount reset button component
+  it.skip("loading should retry loading and keep the menu open when retry button is pressed with Spacebar", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={[]}
+        searchable
+        loading
+        timeout={500}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").invoke(
+      "on",
+      "icRetryLoads",
+      cy.stub().as("icRetryLoad")
+    );
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", "ic-menu ul li ic-typography")
+      .should(BE_VISIBLE)
+      .should("have.text", "Loading Error");
+    cy.get("ic-select").shadow().find(IC_INPUT_CONTAINER);
+  });
+
+  // can't mount clear button component
+  it.skip("should cancel loading if the clear button is pressed mid-load", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={[]}
+        searchable
+        loading
+        timeout={500}
+        showClearButton
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").invoke("on", "icChange", cy.stub().as("icChanges"));
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", "ic-menu ul li ic-typography")
+      .should(BE_VISIBLE)
+      .should("have.text", "Loading Error");
+  });
+
+  it.skip("menu should not be visible on initial load if setting default value and disable filter is set", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={[]}
+        searchable
+        timeout={800}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").shadow().find(IC_MENU_LI).should("not.exist");
+  });
+
+  // can't mount reset button component
+  it.skip("should reset to initial value on form reset with searchable", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={searchableCoffeeOption}
+        searchable
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.get("ic-select").shadow().find(IC_MENU_LI).should(BE_VISIBLE);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_DOWN_ARROW);
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.get(".ic-input").should(HAVE_VALUE, "Lat");
+  });
+
+  it("renders with a default value", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        value="cappuccino"
+        searchable
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.compareSnapshot(
+      "searchable-default-value",
+      DEFAULT_TEST_THRESHOLD + 0.01
+    );
+
+    cy.get("ic-select").shadow().find(IC_INPUT_CONTAINER).type("ca");
```

Comment:
Do we need this line in this test? Instead of it showing that the default value shows results, it tries searching for "cappuccinoca" so the screenshot shows no results found.

Reply:
Agreed. The test and the title is misleading. Will remove this action and update the screenshot below as well.